package.json

```diff
@@ -25,21 +25,21 @@
   "devDependencies": {
     "@extrahorizon/eslint-config-node": "1.6.0",
     "@types/dotenv-safe": "^8.1.2",
-    "@types/jest": "^28.1.3",
+    "@types/jest": "^29.5.11",
     "@types/jsonschema": "^1.1.1",
     "@types/lodash": "^4.14.182",
     "@types/node": "^18.0.0",
     "@types/yargs": "^17.0.10",
     "dotenv-safe": "^8.2.0",
     "eslint": "^8.18.0",
-    "jest": "^28.1.1",
+    "jest": "^29.7.0",
     "mock-fs": "^5.1.2",
     "rimraf": "^3.0.2",
-    "ts-jest": "^28.0.5",
+    "ts-jest": "^29.1.1",
     "typescript": "4.5.5"
   },
   "dependencies": {
-    "@extrahorizon/javascript-sdk": "^7.6.0",
+    "@extrahorizon/javascript-sdk": "^8.0.0",
```

Comment:
As far as I can see, none of the SDK usages is breaking.
The CLI uses a lot of raw calls though 😬

Reply:
Yeah, we needto change that FS, but indeed nothing should be broken with this update 🤷

Reply:
Next time we'll have a test suite to prove it 🤞😄